netmiko/cisco/cisco_xr.py

```diff
@@ -105,36 +114,74 @@ def commit(
         else:
             command_string = "commit"
 
+        if force:
+            command_string = command_string.replace("commit", "commit force")
+        if best_effort:
+            command_string = command_string.replace("commit", "commit best-effort")
+        if replace:
+            command_string = command_string.replace("commit", "commit replace")
+        print(f"command string is {command_string}")
```

Comment:
change to log and add a logger in this file for paramiko logs

Reply:
this was a debugging print, removed it now.